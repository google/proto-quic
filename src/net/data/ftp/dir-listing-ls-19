drwxr-xr-x    2 0        0            4096 Mar 18  2007  
-rw-r--r--    1 48       0         4327486 Jun 16  2006 junorelease.zip
