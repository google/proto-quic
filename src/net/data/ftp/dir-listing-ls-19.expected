-
junorelease.zip
4327486
2006
6
16
0
0
